rancid-tomatillos/src/Dashboard/Dashboard.js

```diff
@@ -1,12 +1,12 @@
 import React from "react";
-import CardsContainer from "./CardsContainer";
+import CardsContainer from "../CardsContainer/CardsContainer";
 
-function MainView(props) {
+function Dashboard(props) {
```

Comment:
Think this was a good call - more industry standard to say "dashboard" instead of main view